docs/src/APIs/Vegetation.md

```diff
@@ -1,34 +1,38 @@
 # Vegetation
 
 ```@meta
-CurrentModule = ClimaLSM.Roots
+CurrentModule = ClimaLSM.Vegetation
 ```
 ## Models
 
 ```@docs
-ClimaLSM.Roots.AbstractVegetationModel
-ClimaLSM.Roots.RootsModel
+ClimaLSM.Vegetation.AbstractVegetationModel
+ClimaLSM.Vegetation.VegetationModel
 ```
 
-## Roots Diagnostic Variables
+## Vegetation Diagnostic Variables
```

Comment:
maybe Plant Hydraulics instead of Vegetation - thinking ahead to when we will have other plant components

Reply:
went with Plant Hydraulics